src/content/en/updates/2019/02/priority-hints.md

```diff
@@ -0,0 +1,432 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+
```

Comment:
Add:

description: Priority Hints provide developers a way to indicate a resource's relative importance to the browser, allowing more control over the order resources are loaded.